tests/apollo/util/bft.py

```diff
@@ -225,23 +233,28 @@ def new(cls, config, client_factory=None):
             builddir=builddir,
             toolsdir=toolsdir,
             procs={},
-            replicas=[bft_config.Replica(i, "127.0.0.1", 3710 + 2*i, 4710 + 2*i)
-                for i in range(0, config.n + config.num_ro_replicas)],
+            replicas=[bft_config.Replica(i, "127.0.0.1",
+                                         bft_config.data_port_from_node_id(i), bft_config.metrics_port_from_node_id(i))
```

Comment:
I'd suggest a better name:
`data_port` -> `bft_port`

Reply:
@teoparvanov, The metrics port isn't bft as well?
I came with this name from previous experience - there is always data port and metrics (usually) and some other ports.
The options are:
1) bft_port
2) bft_main_port
3) bft_msg_port

I like the last option!!
What do you think?

Reply:
`bft_msg_port` sounds good!

Reply:
@teoparvanov ok great. btw, this changes the names of all macros and functions as well (in many places)